UWP Community Toolkit.sln

```diff
@@ -41,6 +41,17 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UnitTests.Notifications.Por
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UnitTests.Notifications.UWP", "UnitTests\UnitTests.Notifications.UWP\UnitTests.Notifications.UWP.csproj", "{BAB1CAF4-C400-4A7F-A987-C576DE63CFFD}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Nuget", "Nuget", "{95EEE926-A52D-4AA1-B3E2-AE16773E39D7}"
```

Comment:
Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Nuget", "Nuget", "{95EEE926-A52D-4AA1-B3E2-AE16773E39D7}"
	ProjectSection(SolutionItems) = preProject
		build\Microsoft.Toolkit.Uwp.Notifications.JavaScript.nuspec = build\Microsoft.Toolkit.Uwp.Notifications.JavaScript.nuspec
		build\Microsoft.Toolkit.Uwp.Notifications.nuspec = build\Microsoft.Toolkit.Uwp.Notifications.nuspec
		build\Microsoft.Toolkit.Uwp.nuspec = build\Microsoft.Toolkit.Uwp.nuspec
		build\Microsoft.Toolkit.Uwp.Services.nuspec = build\Microsoft.Toolkit.Uwp.Services.nuspec
		build\Microsoft.Toolkit.Uwp.UI.Animations.nuspec = build\Microsoft.Toolkit.Uwp.UI.Animations.nuspec
		build\Microsoft.Toolkit.Uwp.UI.Controls.nuspec = build\Microsoft.Toolkit.Uwp.UI.Controls.nuspec
		build\Microsoft.Toolkit.Uwp.UI.nuspec = build\Microsoft.Toolkit.Uwp.UI.nuspec
	EndProjectSection
EndProject

I had added the nuspec files to VS tree